cis-operator.yaml

```diff
@@ -0,0 +1,72 @@
+package:
+  name: cis-operator
+  version: "1.4.1"
+  epoch: 0
+  description: Helps to enable running CIS benchmark security scans on a Kubernetes cluster and generate compliance reports that can be downloaded
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/rancher/cis-operator
+      tag: v${{package.version}}
+      expected-commit: cae64d5d3766e725a29825d4fb7b2c63b62741f1
+
+  - uses: go/bump
+    with:
+      deps: |-
+        github.com/go-viper/mapstructure/v2@v2.3.0
+        golang.org/x/net@v0.38.0
+
+  - uses: go/build
+    with:
+      packages: .
+      tags: netgo,osusergo
```

Comment:
you don't need to specify them here as they are already given as defaults in go/build pipeline: https://github.com/chainguard-dev/melange/blob/5425a36a8e8b21197e45a4f5e019a4963d585970/pkg/build/pipelines/go/build.yaml#L29